### DISCUSSION TITLE
cant set max price

### DISCUSSION THREAD

**Discussion by Swan1360** [jef/streetmerchant#2882] [Help]
street market opens regardless of price even after setting a max

**Comment by Mr-Tech-13** [comment]
Is dotenv called dotnev?

**Reply by Swan1360** [reply]
yea i have it set to best buy and amazon also i have it set only to 3080 and that works but it constantly opens a $1900 gpu from amazon

**Reply by Mr-Tech-13** [reply]
Rename dotnev to dotenv

**Reply by Swan1360** [reply]
its named dotenv

**Comment by Swan1360** [comment]
[https://ibb.co/0qH0ZX6](https://ibb.co/0qH0ZX6)
 [https://ibb.co/hHJ6XLj](https://ibb.co/hHJ6XLj)
 [https://ibb.co/Br566PC](https://ibb.co/Br566PC)

**Comment by Mr-Tech-13** [comment]
Try removing the space between the = and the number
